opm/parser/eclipse/CMakeLists.txt

```diff
@@ -276,6 +276,11 @@ EclipseState/Tables/VFPProdTable.hpp
 EclipseState/Tables/VFPInjTable.hpp
 EclipseState/Tables/TableManager.hpp
 EclipseState/Tables/TableContainer.hpp
+EclipseState/Tables/SorwmisTable.hpp
```

Comment:
The new header `SorwmisTable.hpp` is not part of the PR; so right now it does not compile.